### ISSUE TITLE
Test_Run_Positive_VolumeMountControllerAttachEnabledRace unit failure

### ISSUE THREAD

**Issue by mfojtik** [openshift/origin#21893]
Seen: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/21886/pull-ci-openshift-origin-master-unit/2313/

```
=== RUN   Test_Run_Positive_VolumeMountControllerAttachEnabledRace
E0128 22:42:10.616435   12519 reconciler.go:372] Cannot get volumes from disk open fake-dir: no such file or directory
==================
WARNING: DATA RACE
Write at 0x00c4201117d8 by goroutine 86:
  github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler.Test_Run_Positive_VolumeMountControllerAttachEnabledRace()
      /go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler/reconciler_test.go:1240 +0xf81
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:777 +0x16d
```

**Comment by soltysh** [comment]
I've managed to hit this problem running following script every ~5th-ish iteration of:
```bash
for i in $(seq 1 100); do
	GOTEST_FLAGS="-count=1" TEST_KUBE=true KUBERNETES_SERVICE_HOST= \
            hack/test-go.sh ./vendor/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler/
done
```

**Comment by tsmetana** [comment]
We know about it and I have tried to fix it...

https://github.com/kubernetes/kubernetes/pull/73404

**Comment by sjenning** [comment]
@tsmetana I just hit this again, even after the fix merged
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/22379/pull-ci-openshift-origin-release-3.11-unit/210#githubcomopenshiftoriginvendork8siokubernetespkgkubeletvolumemanagerreconciler-test_run_positive_volumemountcontrollerattachenabledrace

**Comment by sjenning** [comment]
oh wait... that is on the 3.11 branch.  can you pick that fix back to 3.11 branch?

**Comment by sjenning** [comment]
another hit on 3.11 branch https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/23181/pull-ci-openshift-origin-release-3.11-unit/310
